tools/projmgr/src/ProjMgrWorker.cpp

```diff
@@ -1876,7 +1877,7 @@ bool ProjMgrWorker::ListDependencies(vector<string>& dependencies, const string&
   set<string>dependenciesSet;
   for (const auto& selectedContext : m_selectedContexts) {
     ContextItem& context = m_contexts[selectedContext];
-    if (!ProcessContext(context, true, false)) {
+    if (!ProcessContext(context, true, false, false)) {
```

Comment:
Shouldn't the `list dependencies` command also accept and handle the `--no-update-rte` option instead of hard-coding it to `false`?